app/AppKernel.php

```diff
@@ -300,10 +303,18 @@ public function getProjectDir(): string
         return realpath(__DIR__ . '/..');
     }
 
-    protected function getModuleRepository(): ModuleRepository
+    protected function getModuleRepository(): CachedModuleRepository
     {
-        if ($this->moduleRepository === null) {
-            $this->moduleRepository = new ModuleRepository(_PS_ROOT_DIR_, _PS_MODULE_DIR_);
+        if ($this->moduleRepository === null) {//TODO <cnc-notice> DA TESTARE - OK
+            if ($this->environment === 'test') {
```

Comment:
```suggestion
            if ($this->getEnvironment() === 'test') {
```
Actually this would probably be cleaner here